scripts/get-code.js

```diff
@@ -0,0 +1,25 @@
+import 'dotenv/config';
+import fs from 'node:fs';
+
+const candidateId = 374;
+const url = `https://apply.scorchsoft.dev/api/candidate/${candidateId}/get-code`;
+const options = {
+  method: 'POST',
+  body: JSON.stringify({
+    email: 'tsajad300@gmail.com',
```

Comment:
You should always avoid checking credentials into github!

Usually you'd store these in environment variables and load them in here from `process.env`

```
email: process.env.FOOTBALL_EMAIL
password: process.env.FOOTBALL_PASSWORD

```
I think you have a `.env` file which can store the vrariables